packages/sanity/src/core/releases/tool/overview/ReleasesOverview.tsx

```diff
@@ -268,27 +284,37 @@ export function ReleasesOverview() {
     getTimezoneAdjustedDateTimeRange,
   ])
 
+  const RenderCalendarFilter = () => {
+    return (
+      <Flex flex="none">
+        <Card borderRight flex="none" disabled>
+          <CalendarFilter
+            disabled={loading || releases.length === 0}
+            renderCalendarDay={ReleaseCalendarFilterDay}
+            selectedDate={releaseFilterDate}
+            onSelect={handleSelectFilterDate}
+          />
+        </Card>
+      </Flex>
+    )
+  }
```

Comment:
Could we memoize this as a function or move it outside of this function so it's not defined on render?

```suggestion
  const calendarFilter = useMemo(() => {
    return (
      <Flex flex="none">
        <Card borderRight flex="none" disabled>
          <CalendarFilter
            disabled={loading || releases.length === 0}
            renderCalendarDay={ReleaseCalendarFilterDay}
            selectedDate={releaseFilterDate}
            onSelect={handleSelectFilterDate}
          />
        </Card>
      </Flex>
    )
  }, [loading, releases, releaseFilterDate, handleSelectFilterDate])
```

Reply:
I tried doing this but had issues with typing, let me have a look

Reply:
When I tried at the time it must have had something to do with something else (probably typing issues in the props), in any case, I've done this, thank you Pedro!